onfido-java/src/main/models/extracted_data.json

```diff
@@ -0,0 +1,118 @@
+{
+  "title": "ExtractedData",
+  "properties": {
```

Comment:
Again, not sure if we return this in a properties bag? Feel free to ignore if so.

Also: some of these don't seem to match https://documentation.onfido.com/#extraction-result. For example:

`document_type`, `issuing_country` and `issuing_state` seem to be from the `document_classification` breakdown

and I can't see `issuing_date` or `real_id_classification` 

Have to say again that I'm not really in touch with Autofill but it looks like at the least, the reference documentation might need updating which would be good to know.

Reply:
Same again on the properties bit, but I second the rest!

Reply:
Updated, thanks, I copied over from the other branch and just verified all values were present, not if there were any extra lol